auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -567,6 +567,9 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   }
 
   get viewTitle (): string {
+    if (this.isSbcStaffAccount) {
+      return 'My Staff Business Registry'
+    }
```

Comment:
reduce the nested ternary conditions, eg:
```
get viewTitle(): string {
  if (this.isSbcStaffAccount || this.isStaffAccount) {
    return 'My Staff Business Registry'
  }
  return 'My Business Registry'
}
```

Reply:
Request applied.